### PR TITLE
Cyborgs can reset modules in recharging stations

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Cyborgs can reset their module in recharging stations.
/:cl:

When a cyborg is inside a recharging station and has picked a module,
they will be able to reset their module with an action button.

### CODE REVIEW

**Comment by optimumtact** [comment]
:-1: from me, this reduces interactions between robotics and cyborgs

It's very annoying when people do this as it harms the game more than it helps

**Comment by Cyberboss** [comment]
The powercreep is insane

File: code/game/machinery/rechargestation.dm [hunk 1]
```diff
@@ -96,6 +102,7 @@
 /obj/machinery/recharge_station/open_machine()
 	..()
 	use_power = 1
+	reset_module_action.Grant() // empty Grant() removes action from owner
```
Comment (RemieRichards) [inline code/game/machinery/rechargestation.dm]: prefer: ```reset_module_action.Remove(reset_module_action.owner)```

**Comment by KorPhaeron** [comment]
There isn't any silicon power creep, they've been nerfed into near irrelevance lately.

That said borgs are supposed to be specialized, and this flies in the face of that. Don't like it.

**Comment by jughu** [comment]
Just make it so that the robotics have to put a module board in the recharger

**Comment by Cebasiz** [comment]
i like this insofar as it's pretty annoying being a cyborg, going to beg robotics for a module change, then getting renamed to default name because they don't know how to do it, at the same time i'm not sure about it

as for power creep yeah right my man it's not like you can have lmao4secborg on command anymore, oh no some engiborg changes to mediborg to help the station, what a catastrophe that is

tl;dr i think it's worth considering

**Comment by kevinz000** [comment]
@KorPhaeron  "silicons will never again be used for combat by default"

**Comment by optimumtact** [comment]
Hey that's my line

**Comment by PKPenguin321** [comment]
might be worth trying this if it had a yuge cooldown and maybe drained the borgs battery or something

**Comment by Cebasiz** [comment]
yes let's add drawbacks to changing modules so that the nightmare scenario of borgs having something to do doesn't come true

**Comment by Anonmare** [comment]
👎 from me, cyborgs are specialised robots and ought not to be capable of self-improvement without help from someone else.

That being said, if you wanted to make the cyborg chargers a bit more useful - you could make it possible for roboticists to load the docks up with modules and give cyborgs some limited self-service.

**Comment by jughu** [comment]
Just make it like goonstation stick a module board in the recharging station and they get a free module change this way they still require interaction

**Comment by nuklearcellphoneg** [comment]
code red and purged time to find a recharger :^)

**Comment by nuklearcellphoneg** [comment]
Sorry, that might not be relevant to your codebase- my people are currently in diaspora
